CHANGELOG.md

```diff
@@ -26,15 +31,30 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
     - The copyright for all sources is now attributed to Open Enclave SDK contributors.
 - Update Intel DCAP library dependencies to 1.4.1.
 - Update Intel PSW dependencies to 2.6.100.2 on Windows.
-- The stack guard for OE SDK on SGX is turned on. It is also turned on for enclaves.
+- Enable `/W2 /WX` on Windows builds by default to treat W2 warnings as errors.
+- Removed code related to deprecation of strftime.
+- Build OE enclave libraries with stack protector enabled.
```

Comment:
Should this be in added since this is a new feature we are supporting?

Reply:
Done.